website/src/app/views/about/members.js

```diff
@@ -135,6 +135,7 @@ import kyledoliente from '../../../images/about/memberImages/kyledoliente.jpg';
 import kanu from '../../../images/about/memberImages/kanu.jpeg';
 import mazenghosn from '../../../images/about/memberImages/mazenghosn.jpg';
 import hassan from '../../../images/about/memberImages/hassan.jpg';
+import EdWu from '../../../images/about/memberImages/EdWu.jpeg';
```

Comment:
rename "EdWu" to be your first name and last name ("edwardwu")

Reply:
👍